Source/SwiftLintFramework/Rules/Lint/UnusedImportRuleExamples.swift

```diff
@@ -0,0 +1,192 @@
+        """)
+    ]
+
+    static let triggeringExamples = [
+        Example("""
+        ↓import Dispatch
+        struct A {
+          static func dispatchMain() {}
+        }
+        A.dispatchMain()
+        """),
+        Example("""
+        ↓import Foundation // This is unused
+        struct A {
+          static func dispatchMain() {}
+        }
+        A.dispatchMain()
+        ↓import Dispatch
+
+        """),
+        Example("""
+        ↓import Foundation
+        dispatchMain()
+        """),
+        Example("""
+        ↓import Foundation
+        // @objc
+        class A {}
+        """),
+        Example("""
+        ↓import Foundation
+        import UnknownModule
+        func foo(error: Swift.Error) {}
+        """),
+        Example("""
+        ↓import Foundation
+        typealias Foo = CFData
+        """, configuration: [
+            "require_explicit_imports": true
+        ])
+    ]
+
+    static let corrections = [
+        Example("""
+        ↓import Dispatch
+        struct A {
+          static func dispatchMain() {}
+        }
+        A.dispatchMain()
+        """): Example("""
+        struct A {
+          static func dispatchMain() {}
+        }
+        A.dispatchMain()
+        """),
+        Example("""
+        ↓import Foundation // This is unused
+        struct A {
+          static func dispatchMain() {}
+        }
+        A.dispatchMain()
+        ↓import Dispatch
+
+        """): Example("""
+        struct A {
+          static func dispatchMain() {}
+        }
+        A.dispatchMain()
+
+        """),
+        Example("""
+        ↓import Foundation
+        dispatchMain()
+        """): Example("""
+        dispatchMain()
+        """),
+        Example("""
+        ↓@testable import Foundation
+        import Dispatch
+        dispatchMain()
+        """): Example("""
+        import Dispatch
+        dispatchMain()
+        """),
+        Example("""
+        ↓@_exported import Foundation
+        import Dispatch
+        dispatchMain()
+        """): Example("""
+        import Dispatch
+        dispatchMain()
+        """),
+        Example("""
+        ↓import Foundation
+        // @objc
+        class A {}
+        """): Example("""
+        // @objc
+        class A {}
+        """),
+        Example("""
+        @testable import Foundation
+        ↓import Dispatch
+        @objc
+        class A {}
+        """): Example("""
+        @testable import Foundation
+        @objc
+        class A {}
+        """),
+        Example("""
+        @testable import Foundation
+        ↓@testable import Dispatch
+        @objc
+        class A {}
+        """):
+        Example("""
+        @testable import Foundation
+        @objc
+        class A {}
+        """),
+        Example("""
+        ↓↓import Foundation
+        typealias Foo = CFData
+        """, configuration: [
+            "require_explicit_imports": true
+        ], testMultiByteOffsets: false, testOnLinux: false):
```

Comment:
Is it just a coincidence that `testMultibyteOffsets` and `testOnLinux` are always paired in this PR? Is there a scenario where you would want one but not the other?

Reply:
Just a coincidence.